aemedge/blocks/accordion/accordion.css

```diff
@@ -37,64 +35,95 @@
   border-bottom: none;
   border-radius: 4px;
   transform: translate3d(0, 0, 0);
```

Comment:
You can remove all the border + transform attributes, lines 32-37 since it is all transparent or 0.